PKPDsim.Rproj

```diff
@@ -17,5 +17,6 @@ StripTrailingWhitespace: Yes
 
 BuildType: Package
 PackageUseDevtools: Yes
+PackageCleanBeforeInstall: No
```

Comment:
this is the setting I mentioned. For PKPDsim it's better if this is set to `no`, since we don't want to keep compiling the C++ code